node/src/blockchain/blockchain_interface.rs

```diff
@@ -719,14 +798,18 @@ mod tests {
             .into_iter()
             .map(|request| serde_json::from_slice(&request.body()).unwrap())
             .collect();
+        let bodies: Vec<String> = bodies
+            .into_iter()
+            .map(|b| serde_json::to_string(&b).unwrap())
+            .collect();
         assert_eq!(
-            format!("\"0x000000000000000000000000{}\"", &to[2..]),
-            bodies[0]["params"][0]["topics"][2].to_string(),
+            vec![format!("[{{\"id\":0,\"jsonrpc\":\"2.0\",\"method\":\"eth_blockNumber\",\"params\":[]}},{{\"id\":1,\"jsonrpc\":\"2.0\",\"method\":\"eth_getLogs\",\"params\":[{{\"address\":\"0x384dec25e03f94931767ce4c3556168468ba24c3\",\"fromBlock\":\"0x2a\",\"toBlock\":\"0x400\",\"topics\":[\"0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef\",null,\"0x000000000000000000000000{}\"]}}]}}]", &to[2..])],
+            bodies //bodies[0]["params"][0]["topics"][1].to_string(),
```

Comment:
We have a policy of putting the value being asserted on the left side of the `assert_eq!()` macro.

Reply:
So you're saying `actual` then `expected`? I think these escaped the initial policy switch. I think this code was originally written with `expected` then `actual` order.

Reply:
I see. Yeah, sometimes we still come across some old tests where it's reversed. 

Confirming that this is the preferred style: `assert_eq!(actual, expected)`